README.md

```diff
@@ -1 +1,37 @@
-# First-Capstone-Project
\ No newline at end of file
+# First-Capstone-Project
+
+2011 NCTA IELTS Annual Conference San Diego, CA
+
+
+## Mobile version Home page | Desktop version Home page
+![screenshot](img/intoductionScreenshot.png) | ![screenshot](img/introductionDesktopscreenshot.png)
+
+## Mobile version About page | Desktop version About page
+![screenshot](img/aboutpagemobscreen.png) | ![screenshot](img/aboutpagedesktopscreen.png)
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+### Prerequisites
+  * Knowledge on basic html css and javascript
+  * Text Editor (VScode, Atom, Sublime text, etc)
+  * Basic knowledge of node.js
+  * Setup
+  * Clone repository using git clone
+  * Run npm install
+  * Runlive server
+## Built with :
+ * HTML/CSS
+ * Javascript
+ * Linters
+ * Fontawesome icons
+
+ ## Authors
+  * Github:https://github.com/fed1k
+  * LinkedIn:https://www.linkedin.com/in/firdavs-allamurotov-12b60a226/
```

Comment:
[OPTIONAL] Kindly add this as a hyperlink you can do that
```
[text](link)
```
for your Github for example it will be like that [Github](https://github.com/fed1k)